subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        cont = self.sub_stuff["cont"]
+        name_base = cont.get_unique_name()
+        names = []
+        for number in xrange(self.config['names_count']):
+            name = ('%s_%d' % (name_base, number))
+            names.append(name)
+            self.sub_stuff['containers'].append(name)  # just in case
+        self.sub_stuff['subargs'] += ["--name %s" % name for name in names]
+        if self.config['last_name_sticks']:
+            self.sub_stuff['expected_name'] = names[-1]
+        else:
+            self.sub_stuff['expected_name'] = names[0]
+        super(run_tmpfs, self).init_subargs()
+
+    def run_once(self):
+        super(run_tmpfs, self).run_once()
+        cid = self.sub_stuff['cid'] = self.sub_stuff['dkrcmd'].stdout.strip()
+        self.sub_stuff['containers'].append(cid)
+        try:
+            self.sub_stuff["cont"].wait_by_long_id(cid)
```

Comment:
Consider ``self.sub_stuff["cont"].wait_by_name(self.sub_stuff['name'])`` might be easier and there's no reason to use the CID and it's caused trouble elsewhere IIRC.